lib/fastlane/plugin/firebase_app_distribution/actions/login.rb

```diff
@@ -0,0 +1,65 @@
+require 'googleauth'
+require 'googleauth/stores/file_token_store'
+require "google/apis/people_v1"
+# require "googleauth"
+# require "googleauth/stores/file_token_store"
+require "fileutils"
+module Fastlane
+  module Actions
+    module SharedValues
+      TEST_AUTH_CUSTOM_VALUE = :TEST_AUTH_CUSTOM_VALUE
+    end
+
+    class LoginAction < Action
+      def self.run(params)
+        client_id = Google::Auth::ClientId.new("563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com", "j9iVZfS8kkCEFUPaAeJV0sAi")
+        scope = 'https://www.googleapis.com/auth/cloud-platform'
+        token_store = nil
+        authorizer = Google::Auth::UserAuthorizer.new(client_id, scope, token_store)
+        url = authorizer.get_authorization_url(base_url: "urn:ietf:wg:oauth:2.0:oob")
+
+        UI.message("Please open the following address in your browser:")
+        UI.message(url.to_s)
+        UI.message(" ")
```

Comment:
I think an empty string `""` would be better here unless you need the space.

By the way it looks like putting empty UI.message('') calls like this is used within the plugin, which reinforces to me that this was a good approach :+1:: https://github.com/fastlane/fastlane/blob/80a59306fb32b3c29aa5a2e548045c6eb5227f73/fastlane/lib/fastlane/setup/setup.rb#L42